### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by pabs3**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -3109,6 +3135,9 @@ commemmorating->commemorating
 commen->commend, comment, common,
 commend->commend, comment, command,
 commends->commends, comments, commands,
+commenet->comment
+commenetd->commentd
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: type in answer here. should be `commented`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -4496,6 +4568,9 @@ decription->description
 decriptions->descriptions
 decriptor->descriptor
 decriptors->descriptors
+decrmenet->decrement
+decrmenetd->decrementd
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: typo in anwer: should be `decremented

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -6162,6 +6310,7 @@ enrties->entries
 enrty->entry
 ensconsed->ensconced
 entaglements->entanglements
+entended->intended
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: please also add `entend->intend`

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -8216,8 +8385,17 @@ implemenations->implementations
 implemencted->implemented
 implemend->implement
 implemends->implements
+implemenet->implement
+implemenetaion->implementation
+implemenetaions->implementations
 implemenetation->implementation
+implemenetations->implementations
+implemenetd->implementd
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: typo, should be `implemented`

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -8283,6 +8461,16 @@ implimention->implementation
 implimentions->implementations
 implimentor->implementor
 impliments->implements
+implmenet->implement
+implmenetaion->implementation
+implmenetaions->implementations
+implmenetation->implementation
+implmenetations->implementations
+implmenetd->implementd
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: typo

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -8477,11 +8669,16 @@ increament->increment
 increas->increase
 incredable->incredible
 incremeantal->incremental
+incremenet->increment
+incremenetd->incrementd
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: typo

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -8477,11 +8669,16 @@ increament->increment
...
 incremeneted->incremented
 incrementall->incremental, incrementally,
 incremet->increment
 incremetal->incremental
 incrfemental->incremental
+incrmenet->increment
+incrmenetd->incrementd
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: typo

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -8511,6 +8708,7 @@ indempotent->idempotent
 indendation->indentation
 indended->intended, indented,
 indentaion->indentation
+indentended->indented
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: typo

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -14111,6 +14436,10 @@ segementation->segmentation
 segements->segments
 segemnts->segments
 segmantation->segmentation
+segmenet->segment
+segmenetd->segmentd
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: typo

**Comment by pabs3** [comment]
Sorry about the typos, fixed.

I don't think adding entend->intend is a good idea as most of the
occurrences I can find are false positives and entend seems to be a
valid word in some non-English languages.

https://codesearch.debian.net/search?q=%5B%5E%5Cp%7BL%7D%5Dentend%5B%5E%5Cp%7BL%7D%5D

-- 
bye,
pabs

https://bonedaddy.net/pabs3/

**Pull request merged**